anaconda.spec.in

```diff
@@ -99,6 +99,7 @@ Requires: python3-pwquality
 Requires: python3-systemd
 Requires: python3-productmd
 Requires: python3-dasbus >= %{dasbusver}
+Requires: python3-packaging
```

Comment:
This is already required by Blivet, so the new dependency shouldn't be an issue.

Reply:
Perfect, thanks!